tools/distpartitioning/data_shuffle.py

```diff
@@ -292,18 +299,18 @@ def exchange_node_features(rank, world_size, node_feature_tids, ntype_gnid_map,
             own_node_features[feat_key] = []
             own_global_nids[feat_key] = []
             for idx, x in enumerate(output_feat_list):
-                own_node_features.append(x[feat_key])
```

Comment:
is this a bug?

Reply:
why is this bug not captured in the previous test?

Reply:
Yes. This was a bug which made its way into the repo when the code changes were merged from the private copy of the repo in which actual coding and testing are done and when completed these newly added changes are merged into the github repo. From now on, I changed my setup so that all the changes are done on the github repo itself and testing is performed on the same repo. So this will not happen again.